Lesson7

AndreyPushai opened the pull request:
Added completed home tasks for lesson 7.

AndreyPushai committed:
Added lesson7.
Added finished task1.js

AndreyPushai committed:
Upd lesson7

Added completed task2.js

AndreyPushai committed:
Upd lesson7

Added finished task3.js

apolonskiy commented on lesson7/task3.js:
```diff
@@ -0,0 +1,42 @@
+
+function divide(numerator, denominator) {
+    if (denominator === 0) throw new Error("Division by zero.");
+    if (isNaN(numerator) || isNaN(denominator)) 
```
я б не покладався на NaN тут. null, false не будуть NaN і не пройдуть цю перевірку
typeof numerator !== 'number' краще і додатково ще можете і на це що у вас є перевірити)

AndreyPushai replied:
Done.

AndreyPushai committed:
Update task3.js

Added changes to divide function, replaced checks for isNan with typeof value !== 'number'
